packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -125,6 +125,7 @@ class FlutterOptions {
   static const String kUseApplicationBinary = 'use-application-binary';
   static const String kWebBrowserFlag = 'web-browser-flag';
   static const String kWebRendererFlag = 'web-renderer';
+  static const String kGstaticCanvasKitFlag = 'gstatic-canvaskit';
```

Comment:
What do you think of making this less coupled to gstatic and canvaskit? We are aiming to eventually converge on one renderer, at which point referring to "canvaskit" may no longer be meaningful. We may decide to change the CDN to something other than gstatic.com (you never know 🤷 ), so hard-coding "gstatic" in the name may paint us into a corner. It would also be ideal if we had a "one option to rule them all", including custom URLs our users may want to use instead of our defaults.

Strawman:

```
# Full option: specifies a CDN or a custom relative URL, up to the developer.
flutter run --web-static-resources=https://my.favorite.cdn/flutter-web-resources/
flutter run --web-static-resources=/a/different/path/for/resources/

# A special value that instructs us to use bundled resources relative to the base URL (currently `/canvaskit/`).
flutter run --web-static-resources=bundled

# The following two are equivalent and mean "use the default CDN" (currently gstatic.com).
flutter run --web-static-resources=default
flutter run
```

Reply:
How about `--web-resources-cdn`? I agree we should make it more general than "gstatic CanvasKit" but I think we should have the configuration object/FLUTTER_WEB_CANVASKIT_URL be the way to override the static resource URL and this flag is just a boolean flag saying "please set the resource base URL to the CDN you have set up".

Reply:
SGTM!